.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# This CODEOWNERS file sets the individuals responsible for code in the zksync-hardhat-template repository.
+
+# These users are the default owners for everything in the repo.
+# They will be requested for review when someone opens a pull request.
+* @matter-labs/devxp
```

Comment:
I'm hesitant to add the `.github/` directory, as this is a template repo that others are expected to copy/paste its contents.

At the very least, we shouldn't merge this PR until we update `zksync-cli` accordingly: https://github.com/matter-labs/zksync-cli/blob/main/src/commands/create/create.ts#L76-L80

Reply:
Ah, right, will work on that

Reply:
Should be fixed in https://github.com/matter-labs/zksync-cli/pull/73

Reply:
Fixed!